### ISSUE TITLE
PvP does not change even when the command works

### ISSUE THREAD

**Issue by TheRealDadbeard** [K-4U/pvpToggle#14]
Newest version on forge 2659. It just does nothing but says it changed. Anyway do reset the whole config where the pvp state is saved maybe that can fix it?

**Comment by K-4U** [comment]
Morning.

What do you mean by "Anyway do reset the whole config where the pvp state is saved maybe that can fix it?"?

You need to enable pvp in your servers configuration. The mod will take care of it further.

**Comment by Swizzler121** [comment]
I think they mean if a player is set to enabled or disabled since that carries over between server reboots and disconnects, there's got to be a way to delete that, unless it's stored in the playerdata?

**Comment by TheRealDadbeard** [comment]
Yeah sorry bad language. I meant that when I use the command nothing changes at all. The icon stays the same and Ingame PVP stays off even when both player switch.

Edit: Looked at the pvptoggle json file. Nothing changes there. Still on false.
